### PR TITLE
Feat(eos_designs): Allow LACP timers configuration under connected_endpoints

### LINKED ISSUE #2805: Allow LACP fast timer for endpoints in eos_designs
### Enhancement summary

Allow LACP configuration on eos_designs for defined connected endpoints.

### Which component of AVD is impacted

eos_designs

### Use case example

Specific networking devices to use a fast LACP timer

### Describe the solution you would like

Add a new keyword under the port-channel adapter in eos_designs:
```
lacp_timer:
  fast: < bool > | default is false
  multiplier: < number of missed PDUs to time out a member> | Optional - default is 3
```

It should generate the required configuration under each ethernet port in the port-channel

### Describe alternatives you have considered

_No response_

### Additional context

_No response_

### Contributing Guide

- [X] I agree to follow this project's Code of Conduct

### PR DESCRIPTION
**Pull request by xaviramon**
## Change Summary

This PR allows configuration of LACP timers for adapters under connected-endpoints, allowing for LACP interaction to endpoints.

## Related Issue(s)

Fixes #2805 

## Component(s) name

`arista.avd.eos_desings`

## Proposed changes
New data model inside the port-channel definition of an adapter:
```
lacp_timer:
    mode: <normal | fast>
    multiplier: <integer>
```
This datamodel is read in the ethernet interfaces and applied to the port-channel because configuration of this feature requires a port-channel using LACP (mode on is ignored) and the configuration in the CLI happens at the ethernet interface itself.

## How to test
Molecule included

### User Checklist

- N/A

### Repository Checklist

- [x] My code has been rebased from devel before I start
- [x] I have read the [**CONTRIBUTING**](https://avd.sh/en/latest/docs/contribution/overview.html) document.
- [x] My change requires a change to the documentation and documentation have been updated accordingly.
- [x] I have updated [molecule CI](https://github.com/aristanetworks/ansible-avd/tree/devel/ansible_collections/arista/avd/molecule) testing accordingly. (check the box if not applicable)

### CODE REVIEW

File: ansible_collections/arista/avd/roles/eos_designs/schemas/schema_fragments/defs_adapter_config.schema.yml [hunk 1]
```diff
@@ -456,6 +456,21 @@ $defs:
                 convert_types:
                   - str
                 description: Timeout in seconds, Optional - default is 90 seconds.
+          lacp_timer:
+            type: dict
+            description: LACP timer configuration.
```
Comment (gmuloc) [inline ansible_collections/arista/avd/roles/eos_designs/schemas/schema_fragments/defs_adapter_config.schema.yml]: Add in desc that it is configured only if `mode != 'on'`

File: ansible_collections/arista/avd/molecule/eos_designs_unit_tests/inventory/group_vars/DC1_SERVERS.yml [hunk 2]
```diff
@@ -552,6 +562,30 @@ servers:
...
+    adapters:
+      - endpoint_ports: [ Eth1, Eth2 ]
+        switch_ports: [ Ethernet32, Ethernet32 ]
+        switches: [ DC1-SVC3A, DC1-SVC3B ]
+        port_channel:
+          description: server25_port_channel_with_lacp_timer
+          mode: active
+          enabled: true
+          lacp_timer:
+            mode: fast
```
Comment (gmuloc) [inline ansible_collections/arista/avd/molecule/eos_designs_unit_tests/inventory/group_vars/DC1_SERVERS.yml:596]: try the normal mode in one of the tests

**Review by ClausHolbechArista — APPROVED** [review]

File: ansible_collections/arista/avd/roles/eos_designs/schemas/schema_fragments/defs_adapter_config.schema.yml [hunk 3]
```diff
@@ -458,7 +458,7 @@ $defs:
                 description: Timeout in seconds, Optional - default is 90 seconds.
           lacp_timer:
             type: dict
-            description: LACP timer configuration.
+            description: LACP timer configuration. Optional - Applies only when Port-channel mode is not "on".
```
Comment (gmuloc) [inline ansible_collections/arista/avd/roles/eos_designs/schemas/schema_fragments/defs_adapter_config.schema.yml]:
```suggestion
            description: LACP timer configuration. Applies only when Port-channel mode is not "on".
```

File: ansible_collections/arista/avd/roles/eos_designs/schemas/schema_fragments/defs_adapter_config.schema.yml [hunk 4]
```diff
@@ -456,6 +456,21 @@ $defs:
...
+                type: str
+                valid_values:
+                  - normal
+                  - fast
+                description: LACP mode for interface members.
+              multiplier:
+                type: int
+                convert_types:
+                  - str
+                description: Number of BPDUs lost to time out a member from the Port-channel, Optional - default is 3.
```
Comment (ClausHolbechArista) [inline ansible_collections/arista/avd/roles/eos_designs/schemas/schema_fragments/defs_adapter_config.schema.yml]:
```suggestion
                description: Number of LACP BPDUs lost before deeming the peer down. EOS default is 3.
```

File: ansible_collections/arista/avd/roles/eos_designs/schemas/schema_fragments/defs_adapter_config.schema.yml [hunk 5]
```diff
@@ -429,7 +429,7 @@ $defs:
             type: bool
             default: true
             description: |
-              Port-Channel administrative state, Optional.
+              Port-Channel administrative state.
               setting to false will set port to 'shutdown' in intended configuration.
```
Comment (ClausHolbechArista) [inline ansible_collections/arista/avd/roles/eos_designs/schemas/schema_fragments/defs_adapter_config.schema.yml]:
```suggestion
              Setting to false will set port to 'shutdown' in intended configuration.
```

**Review by ClausHolbechArista — APPROVED** [review]

**Review by gmuloc — APPROVED** [review]

**Pull request merged**